README.adoc

```diff
@@ -68,17 +62,20 @@ mvn install liberty:start-server
 
 Maven builds the application and runs it inside Open Liberty.
 
-Point your browser to the `http://localhost:9080/inventory/systems` URL.
-Because you just started the application, the inventory is currently empty. Point to the
+Point your browser to the `http://localhost:9080/inventory/systems` URL to access the `inventory`
+service. Because you just started the application, the inventory is currently empty. Then, access the
 `http://localhost:9080/inventory/systems/localhost` URL to add the localhost host into the inventory.
 
-Next, point to the `https://localhost:9443/metrics` MicroProfile Metrics URL. Click the `Advanced`
-button to proceed. Use the `confAdmin` and `microprofile` credentials to log in. You can see both
-the standard metrics and the application metrics in a text format.
+Access the `inventory` service i.e. the `http://localhost:9080/inventory/systems` URL at least once
```

Comment:
We want to avoid Latin phrases and abbreviations like "i.e.", especially if we ever want to translate the guides.
What about "Access the `inventory` service, such as the `http://localhost:9080/inventory/systems` URL, at least..."
or
"Access the `inventory` service at the `http://localhost:9080/inventory/systems` URL at least..."